src/databrowser/core.py

```diff
@@ -628,7 +628,9 @@ async def _iterintake(self) -> AsyncIterator[str]:
             _, results = res
             for out in results.get("response", {}).get("docs", [{}]):
                 source = {
-                    k: out[k]
+                    k: out[k][0]
+                    if isinstance(out.get(k), list) and len(out.get(k)) == 1
+                    else out.get(k)
```

Comment:
Ok, now I get it. But is the og problem here or is this just a fix to another problem? As in, why is there a list if it's supposed to be a string? Did something go wrong when crawling metadata? A solr config issue?

Anyhoo I think this is fair enough!

Reply:
I think the reason why (almost) everything is a list is because of the way how our Solr schema looks like, as almost everything is a single-element-list in there. I guess I am the one to blame for this decision as I had no idea how the data we ingest look like and just decided to make everything a list in the schema "just-in-case" so that nothing crashes.

It worked for the first ~150 Elements because you did the exact same thing I am doing in this PR for them in the initial Solr-call.